### PR TITLE
[frontend] Fix double click on add/update opinion in entities

### PR DESCRIPTION
**Pull request by RomuDeuxfois**
### Need help

* I try to fix this issues for the second time. I am not able to reproduce in local, so it's complicated to find the fix. If someone have some ideas to help me on it, I would appreciate.

### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/3838

### CODE REVIEW

**Comment by yassine-ouaamou** [comment]
I will try to reproduce

File: opencti-platform/opencti-front/src/private/components/analyses/opinions/StixCoreObjectOpinionsRadar.tsx [hunk 1]
```diff
@@ -182,10 +182,10 @@ StixCoreObjectOpinionsRadarProps
           input: finalValues,
         },
         onCompleted: () => {
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/analyses/opinions/StixCoreObjectOpinionsRadar.tsx:185]: updater: () => fetchQuery(),
        onCompleted: () => {
          setSubmitting(false);
          resetForm();
        },

File: opencti-platform/opencti-front/src/private/components/analyses/opinions/StixCoreObjectOpinionsRadar.tsx [hunk 2]
```diff
@@ -195,10 +195,10 @@ StixCoreObjectOpinionsRadarProps
           input: inputValues,
         },
         onCompleted: () => {
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/analyses/opinions/StixCoreObjectOpinionsRadar.tsx:197]: updater: () => fetchQuery(),
        onCompleted: () => {
          setSubmitting(false);
          resetForm();
        },

**Review by Kedae — CHANGES_REQUESTED** [review]

**Comment by Kedae** [comment]
Testable with slow 3G throttling.

The issue is that we fetch the query (i.e try to re-render the component) while mutating the relay store = Kaboom.
Updater function is called when the local relay store HAS been updated, not while completion.

**Review by Kedae — DISMISSED** [review]

**Comment by yassine-ouaamou** [comment]
I still reproduce this bug using the 3G throttling
throttling

**Comment by SarahBocognano** [comment]
Tested: Still happening is Slow 3G, we first have the "create" button appearing, then when clicking, the "update" button finally appears"

**Comment by SouadHadjiat** [comment]
I still have the bug too with "fast 3G" or less. The dialog keeps displaying even though the "open" variable is false. You can't close it with "Cancel" button, you need to update again or change any value. The only way I found to close the dialog is to close it at the beggining of `onSubmit`, because it looks like in `onComplete `or `updater`, we don't really have the same scope anymore, because of the refetch query.

**Pull request closed without merging**